app/Filament/Clusters/ServiceManagementAdministration.php

```diff
@@ -44,7 +44,7 @@ class ServiceManagementAdministration extends Cluster
 
     protected static ?string $navigationGroup = 'Product Administration';
 
-    protected static ?int $navigationSort = 6;
+    protected static ?int $navigationSort = 60;
```

Comment:
It looks like we have a duplicated sort of 60 between this and `app/Filament/Clusters/AssetManagement.php`. Can you adjust this, and bump up every item after this one?

Reply:
> It looks like we have a duplicated sort of 60 between this and `app/Filament/Clusters/AssetManagement.php`. Can you adjust this, and bump up every item after this one?

HI,

Issue is fixed.

Thank you.